user-docs/latest/content-manager/configuring-view-of-content-type.md

```diff
@@ -43,3 +59,56 @@ Note also that relational fields have a couple limitations when it comes to sort
 - Sorting cannot be enabled for relational fields which display several fields.
 - Relational fields cannot be set as default sort.
 :::
+
+## Configuring the edit view
+
+In the edit view of a content type, in the right side of the interface, a **Configure the view** button is displayed. It allows to access the configurations that can be set for the edit view of the content type, such as the entry title, and the display of the fields of the content type, including the relational ones.
+
+![Configuring the edit view of the Content Manager](../assets/content-manager/edit-view-config.png)
+
+### Edit view settings
+
+1. In the edit view of your content type, click on the **Configure the view** button.
+2. In the Settings area, define your chosen new settings:
+
+| Setting name    | Instructions                                                                          |
+| --------------- | ------------------------------------------------------------------------------------- |
+| Entry title     | Choose among the drop-down list the field that should be used as title for the entry. |
+
+3. Click on the **Save** button.
+
+### Edit view display
+
+1. In the edit view of your content type, click on the **Configure the view** button.
+2. In the View area, define what fields to display in the list view table, and in what order:
+   - Click the add button ![icon add new](../assets/content-manager/icon_add.png) to add a new field.
+   - Click the delete button ![icon delete](../assets/content-manager/icon_delete.png) to remove a field.
+   - Click the reorder button ![icon drag & drop](../assets/content-manager/icon_dragdrop.png) and drag and drop it to the place you want it to be displayed amond the other fields.
+3. Click the name of a field to access its available own settings:
+
+| Setting name    | Instructions                                                                              |
+| --------------- | ----------------------------------------------------------------------------------------- |
+| Label           | Write the label to should be used for the field.                                          |
+| Description     | Write a description for the field, to help other administrators fill it properly.         |
+| Placeholder     | Write the placeholder that should be displayed by default in the field.                   |
+| Editable field  | Click on **ON** or **OFF** to able or disable the edition of the field by administrators. |
+
+4. Click on the **Save** button.
+
+::: warning IMPORTANT
+The settings and display of a component's fields cannot be managed and reordered through the entry's edit view configuration page. Click on the **Set the component's layout** button of a component to access the component's own configuration page. You will find the exact same settings and display options as for the entry, but that will specifically apply to your component.
+:::
+
+#### Relational fields
```

Comment:
In this section, you are using `regular fields` to define non relational fields.
Here you are using a title to introduce `relational fields`, why not using one also for `regular fields`.
And so, having the same content structure.
Here in I can see the procedure is with a bullet point instead of a ordered list.
And the format is different compared to `regular fields`.
Same the settings format is different, for the entry title.

I think I would have the same structure for `regular fields` and `relational fields`

Reply:
I can't have the same structure for both otherwise too much pieces of information will have to be repeated, which is something to avoid. That's why I chose to keep a similar structure for Edit view and List view configurations, and add an extra sub-section for relational fields in Edit view configurations. 
Also, the bullet list is not a procedure. I can however see why it's confusing, so I'll improve the content while keeping the structure I think.

Reply:
I agree @meganelacheny the duplication of content is something we should avoid